tests/linux_benchmarks/iperf_benchmark_test.py

```diff
@@ -0,0 +1,145 @@
+# Copyright 2020 PerfKitBenchmarker Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for netperf_benchmark."""
+
+import json
+import os
+import unittest
+from absl import flags
+import mock
+import itertools
+
+from perfkitbenchmarker import benchmark_spec
+from perfkitbenchmarker import errors
+from perfkitbenchmarker import vm_util
+from perfkitbenchmarker.linux_benchmarks import iperf_benchmark
+
+FLAGS = flags.FLAGS
+
+class IperfBenchmarkTestCase(unittest.TestCase):
+
+  def testIperfParseResults(self):
+
+    vm_spec = mock.MagicMock(spec=benchmark_spec.BenchmarkSpec)
+    vm0 = mock.MagicMock()
+    vm1 = mock.MagicMock()
+    vm_spec.vms = [vm0, vm1]
+    path = os.path.join(os.path.dirname(__file__), '..', 'data', 'iperf_results.txt')
+    outfile = open(path, 'r')
```

Comment:
Rename "path" to "iperf_results_file" and use the context manager way to read files:
  with open(iperf_results_file) as reader:
      iperf_stdout_list = reader.read().split('#')

Reply:
I went with @dlott's suggestion above and just included the outputs in the test file as triple quoted strings